Makefile

```diff
@@ -36,6 +36,8 @@ test:
 generate-stubs:
 	uv run stubgen -p logfire --include-docstrings --no-analysis
 	rsync -a out/logfire/ logfire-api/logfire_api/
+	uv run pytest ./tests/test_logfire_api.py::test_override_init_pyi || true
```

Comment:
`|| true` so that we ignore the test failure on the first pass, it should report as skipped on the second

Reply:
this should be a comment in the code